libraries/AP_Math/location.h

```diff
@@ -7,7 +7,9 @@
 
 #include "vector2.h"
 #include "vector3.h"
+#include <AP_Common/Location.h>
```

Comment:
This looks odd.  Predeclaring Location *after* including the header?!

Reply:
removed